include/CMakeLists.txt

```diff
@@ -16,7 +16,8 @@ target_sources(${LRS_TARGET}
         "${CMAKE_CURRENT_LIST_DIR}/librealsense2/h/rs_internal.h"
         "${CMAKE_CURRENT_LIST_DIR}/librealsense2/h/rs_pipeline.h"
         "${CMAKE_CURRENT_LIST_DIR}/librealsense2/h/rs_config.h"
-        "${CMAKE_CURRENT_LIST_DIR}/librealsense2/h/rs_terminal_parser.h"
+        "${CMAKE_CURRENT_LIST_DIR}/librealsense2/h/rs_firmware_logs.h"
```

Comment:
Do we want separate files? @ev-mp why not put into rs-internal?

Reply:
Sounds good, initially the proposed API was more verbose.